src/vm/arm64/stubs.cpp

```diff
@@ -2199,6 +2244,22 @@ PCODE DynamicHelpers::CreateDictionaryLookupHelper(LoaderAllocator * pAllocator,
                 p += 4;
                 dataOffset -= 4; // subtract 4 as we have moved PC by 4
             }
+
+            if (i == 0 && pLookup->indirectFirstOffset)
+            {
+                // 12 bytes
+                // add x0, x0, x9
+                *(DWORD*)p = 0x8b090000;
+                p += 4;
+
+                // ldr x9, [sp], 0x10
+                *(DWORD*)p = 0xf84107e9;
+                p += 4;
+
+                // add sp, sp, 0x10
+                *(DWORD*)p = 0x910043ff;
```

Comment:
This looks unnecessary becuase you claim to be using post index form above.Use post index form